transposer/plugins/org.polarsys.kitalpha.transposer.rules.handler.mapping.edit/generated/org/polarsys/kitalpha/transposer/rules/handler/rules/common/provider/MappingItemProvider.java

```diff
@@ -503,17 +529,19 @@ public void notifyChanged(Notification notification) {
 	protected void collectNewChildDescriptors(Collection<Object> newChildDescriptors, Object object) {
 		super.collectNewChildDescriptors(newChildDescriptors, object);
 
-				newChildDescriptors.add
-					(createChildParameter
-						(CommonPackage.Literals.MAPPING__OWNED_PACKAGES,
-						 CommonFactory.eINSTANCE.createMappingPackage()));
+
+		newChildDescriptors.add
+			(createChildParameter
+				(CommonPackage.Literals.MAPPING__OWNED_PACKAGES,
+				 CommonFactory.eINSTANCE.createMappingPackage()));
+
 
 
 
-				newChildDescriptors.add
-					(createChildParameter
-						(CommonPackage.Literals.MAPPING__OWNED_MAPPING_ELEMENTS,
-						 CommonFactory.eINSTANCE.createMappingElement()));
+		newChildDescriptors.add
+			(createChildParameter
+				(CommonPackage.Literals.MAPPING__OWNED_MAPPING_ELEMENTS,
+				 CommonFactory.eINSTANCE.createMappingElement()));
```

Comment:
This is not really a modification, only a reformatting. Please undo (there are more of these).

Reply:
the same as above

Reply:
I cannot undo modifications on this file because there is several adding according to the new feature: for instance: 

getString("_UI_PropertyDescriptor_description", "_UI_Mapping_selectableOutputFolder_feature", "_UI_Mapping_type"), //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
				 CommonPackage.Literals.MAPPING__SELECTABLE_OUTPUT_FOLDER,
				 true,
				 false,
				 false,